c2rust-analyze/src/rewrite/expr/convert.rs

```diff
@@ -594,6 +707,27 @@ fn generate_zeroize_code(zero_ty: &ZeroizeType, lv: &str) -> String {
     }
 }
 
+/// Generate an expression to produce a zeroized version of a value.
+fn generate_zeroize_expr(zero_ty: &ZeroizeType) -> String {
```

Comment:
Couldn't you use `Default::default` here? Is it because the trait isn't implemented for generic-sized arrays?

Reply:
It's also not implemented for c2rust-transpiled structs